gluten-ut/spark33/src/test/scala/org/apache/spark/sql/GlutenBloomFilterAggregateQuerySuite.scala

```diff
@@ -45,4 +45,36 @@ class GlutenBloomFilterAggregateQuerySuite
       }
     }
   }
+
+//  override def sparkConf: SparkConf = {
+//    super.sparkConf
+//      .set("spark.gluten.sql.columnar.backend.ch.runtime_config.logger.level", "debug")
+//      .setMaster("local[1]")
+//  }
```

Comment:
remove？

Reply:
i left it on purpose for convenience of debugging.

Reply:
Agree with @zzcclp . We'd better remove this part.